js/components/updateBar.js

```diff
@@ -92,10 +92,10 @@ class UpdateAvailable extends ImmutableComponent {
       }
       <Button className='updateButton updateLaterButton updateSecondaryButton'
         l10nId='updateLater'
-        onClick={appActions.setUpdateStatus.bind(null, UpdateStatus.UPDATE_AVAILABLE_DEFERRED, false)} />
+        onClick={appActions.setUpdateStatus.bind(null, UpdateStatus.UPDATE_AVAILABLE_DEFERRED, false, undefined)} />
       <Button className='updateButton updateNowButton'
         l10nId='updateNow'
-        onClick={appActions.updateRequested} />
+        onClick={appActions.setUpdateStatus.bind(null, UpdateStatus.UPDATE_APPLYING_RESTART, false, undefined)} />
```

Comment:
We're moving away from having bind in the handler because react sees it as different every time.  Preferred method is to add a method and then call it and bind it to this in the constructor.